Makefile

```diff
@@ -27,6 +27,9 @@ uninstall:
 test: setup.bin build
 	@./setup.bin -test
 
+rrdd/version.ml: VERSION
```

Comment:
Build fails because of this:
`- make: *** No rule to make target `VERSION', needed by `rrdd/version.ml'.  Stop.`